docs/iris/src/userguide/interpolation_and_regridding.rst

```diff
@@ -96,9 +96,9 @@ It isn't necessary to specify sample points for every dimension, only those that
 wish to interpolate over:
 
     >>> result = air_temp.interpolate([('longitude', 0)], iris.analysis.Linear())
-    >>> print 'Original:', air_temp.summary(shorten=True)
+    >>> print('Original: ' + str(air_temp.summary(shorten=True)))
     Original: air_temperature / (K)               (latitude: 73; longitude: 96)
-    >>> print 'Interpolated:', result.summary(shorten=True)
+    >>> print('Interpolated: ' + str(result.summary(shorten=True)))
```

Comment:
Plus, I wasn't sure where to put it. But maybe it will work in the `doctest_global_setup`option that I just found.